react/week3/TodoDescription.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+
+function TodoDescription({ addTodo }) {
+  const [state, setState] = useState("");
+  const [date, setDate] = useState(new Date());
+
+  const onFormSubmit = event => {
+    event.preventDefault();
+    if (!state || !date) {
+      return alert("Please enter details");
+    }
+    addTodo(state, date.toISOString().substr(0, 10));
+    setState("");
+    setDate("");

```

Comment:
In general is good practice to reset the state to the initial state
```suggestion
    setDate(new Date());

```